src/components/QuickOpenModal.js

```diff
@@ -170,10 +169,9 @@ export class QuickOpenModal extends Component<Props, State> {
     }
 
     if (searchType === "variables") {
-      selectSource(selectedSource.get("id"), {
-        location: {
-          ...(item.location != null ? { line: item.location.start.line } : {})
-        }
+      selectLocation({
+        sourceId: selectedSource.get("id"),
+        ...(item.location != null ? { line: item.location.start.line } : null)
```

Comment:
same comment here! a bit too much going on in this line

Reply:
Was trying to be smart instead of being clear. Thanks.